consensus/istanbul/config.go

```diff
@@ -81,7 +98,28 @@ var DefaultConfig = &Config{
 	AnnounceAdditionalValidatorsToGossip:           10,
 }
 
-type ProxyConfig struct {
-	InternalNode *enode.Node `toml:",omitempty"` // The internal facing node of the proxy that this proxied validator will peer with
-	ExternalNode *enode.Node `toml:",omitempty"` // The external facing node of the proxy that the proxied validator will broadcast via the announce message
+//ApplyChainConfigToConfig applies chainConfig values to Config
+func ApplyChainConfigToConfig(chainConfig *params.ChainConfig, config *Config) error {
```

Comment:
```suggestion
//ApplyParamsChainConfigToConfig applies the istanbul config values from params.chainConfig to the istanbul.Config config
func ApplyParamsChainConfigToConfig(chainConfig *params.ChainConfig, config *Config) error {
```

The word "config" is in there like 10,000 times, but I think given two different config types are involved it's better to call it out in the commend and function name.  Would need to be updated where the function is called, as well.